spec/PeriodicBackgroundSync-index.bs

```diff
@@ -24,6 +24,7 @@ spec:web-background-sync;
     type:dfn; text:online
     type:dfn; text:in the background
 spec:permissions-1; type:dict-member; text:name
+spec:infra; type:dfn; text:list
```

Comment:
you don't need this if you just use list as: [=/list=]

Reply:
done